app/models/activity.rb

```diff
@@ -28,3 +28,25 @@ def readonly?
     !new_record?
   end
 end
+
+# == Schema Information
```

Comment:
Schema info in models - great tip, but its not the topic of PR
I will appreciate if you open another PR just with this stuff